src/ManifoldsBase.jl

```diff
@@ -542,19 +575,23 @@ By default these informations are collected by calling [`check_approx`](@ref).
 Keyword arguments can be used to specify tolerances.
 """
 function isapprox(M::AbstractManifold, p, X, Y; error::Symbol = :none, kwargs...)
-    mat = check_approx(M, p, X, Y; kwargs...)
-    if mat !== nothing
-        (error === :error) && throw(mat)
-        if isnan(mat.val)
-            s = "$(typeof(mat))\n$(mat.msg)"
-        else
-            s = "$(typeof(mat)) with $(mat.val)\n$(mat.msg)"
+    if error === :none
+        return isapprox(X, Y; kwargs...)
```

Comment:
same as above, just a bit worse, since equality of two  vectors might depend on the point?

Reply:
This is similar thing to point comparison, let's discuss it there.